### PR TITLE
migrate CFS to AKS

### PR DESCRIPTION
**Pull request by yarpo**
Migrate CFS to AKS.

This was meant to be extracted into a lib, but we didn't have enough time to test this solution so we now need to migrate into AKS in terms of creating AKS demo env.

### CODE REVIEW

File: k8s/aat/cluster-00/divorce/div-cfs.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      memoryLimits: '2048Mi'
+      cpuLimits: '1500m'
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
+      image: hmctspublic.azurecr.io/div/cfs:prod-f3004785
+      environment:
+        REFORM_SERVICE_NAME: "cfs"
```
Comment (timja) [inline k8s/aat/cluster-00/divorce/div-cfs.yaml]: you shouldn't need any of these vars. i doubt they do anything

File: k8s/aat/cluster-00/divorce/div-cfs.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        REFORM_TEAM: "div"
+        SLOT: "PRODUCTION"
+      aadIdentityName: divorce
+      prometheus:
+        enabled: true
+    global:
+      environment: aat
+      subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
```
Comment (timja) [inline k8s/aat/cluster-00/divorce/div-cfs.yaml]:
```suggestion
      subscriptionId: "remove after updating base java chart"
```

File: k8s/aat/cluster-00/divorce/div-cfs.yaml (hunk 1 above)
Comment (yarpo) [reply k8s/aat/cluster-00/divorce/div-cfs.yaml]: ok

File: k8s/aat/cluster-00/divorce/div-cfs.yaml (hunk 2 above)
Comment (yarpo) [reply k8s/aat/cluster-00/divorce/div-cfs.yaml]: we have:

```dependencies:
  - name: java
    version: ~2.11.2
    repository: "@hmctspublic"
```

**Pull request closed without merging**